PyInstaller/utils/hooks/gi.py

```diff
@@ -149,56 +145,43 @@ def gir_library_path_fix(path):
 
         # g-ir-compiler expects a file so we cannot just pipe the fixed file to it.
         command = subprocess.Popen((
-            'g-ir-compiler', os.path.join(CONF['workpath'], gir_name),
-            '-o', os.path.join(CONF['workpath'], typelib_name)
-        ))  # yapf: disable
+            'g-ir-compiler', os.path.join(CONF['workpath'],
+                                          gir_name), '-o', os.path.join(CONF['workpath'], typelib_name)
+        ))
```

Comment:
This reverts fixes from formatting commit. I'm not particularly fond of default splitting that yapf does here.